code/modules/ruins/lavalandruin_code/elephantgraveyard.dm

```diff
@@ -0,0 +1,247 @@
+	else
+		new /obj/item/stack/sheet/bone(src, 1)
+
+//***Oil well puddles.
+/obj/structure/sink/oil_well	//You're not going to enjoy bathing in this...
+	name = "oil well"
+	desc = "A bubbling pool of oil.This would probably be valuable, had bluespace technology not destroyed the need for fossil fuels 200 years ago."
+	icon = 'icons/obj/watercloset.dmi'
+	icon_state = "puddle-oil"
+	dispensedreagent = /datum/reagent/oil
+
+/obj/structure/sink/oil_well/Initialize()
+	.=..()
+	create_reagents(20)
+	reagents.add_reagent(dispensedreagent, 20)
+
+/obj/structure/sink/oil_well/attack_hand(mob/M)
+	flick("puddle-oil-splash",src)
+	reagents.reaction(M, TOUCH, 20) //Covers target in 20u of oil.
+	to_chat(M, "<span class='notice'>You touch the pool of oil, only to get oil all over yourself. It would be wise to wash this off with water.</span>")
+
+/obj/structure/sink/oil_well/attackby(obj/item/O, mob/user, params)
+	flick("puddle-oil-splash",src)
+	if(O.tool_behaviour == TOOL_SHOVEL && !(flags_1&NODECONSTRUCT_1)) //attempt to deconstruct the puddle with a shovel
+		to_chat(user, "You fill in the oil well with soil.")
+		O.play_tool_sound(src)
+		deconstruct()
+		return 1
+	if(istype(O, /obj/item/reagent_containers)) //Refilling bottles with oil
+		var/obj/item/reagent_containers/RG = O
+		if(RG.is_refillable())
+			if(!RG.reagents.holder_full())
+				RG.reagents.add_reagent(dispensedreagent, min(RG.volume - RG.reagents.total_volume, RG.amount_per_transfer_from_this))
+				to_chat(user, "<span class='notice'>You fill [RG] from [src].</span>")
+				return TRUE
+			to_chat(user, "<span class='notice'>\The [RG] is full.</span>")
+			return FALSE
+	if(user.a_intent != INTENT_HARM)
+		to_chat(user, "<span class='notice'>You won't have any luck getting \the [O] out if you drop it in the oil.</span>")
+		return 1
+	else
+		return ..()
+
+/obj/structure/sink/oil_well/drop_materials()
+	new /obj/effect/decal/cleanable/oil(loc)
+
+//***Grave mounds.
+/obj/structure/closet/crate/grave
+	name = "burial mound"
+	desc = "An marked patch of soil, showing signs of a burial long ago. You wouldn't disturb a grave... right?"
+	icon = 'icons/obj/crates.dmi'
+	icon_state = "grave"
+	dense_when_open = TRUE
+	material_drop = /obj/item/stack/ore/glass/basalt
+	material_drop_amount = 5
+	anchorable = FALSE
+	anchored = TRUE
+	locked = TRUE
+	breakout_time = 900
+	cutting_tool = /obj/item/shovel
+	var/lead_tomb = FALSE
+	var/first_open = FALSE
+
+/obj/structure/closet/crate/grave/PopulateContents()  //GRAVEROBBING IS NOW A FEATURE
+	..()
+	new /obj/effect/decal/remains/human/grave(src)
+	switch(rand(1,8))
+		if(1)
+			new /obj/item/coin/gold(src)
+			new /obj/item/storage/wallet(src)
+		if(2)
+			new /obj/item/clothing/glasses/meson(src)
+		if(3)
+			new /obj/item/coin/silver(src)
+			new /obj/item/shovel/spade(src)
+		if(4)
+			new /obj/item/storage/book/bible/booze(src)
+		if(5)
+			new /obj/item/clothing/neck/stethoscope(src)
+			new	/obj/item/scalpel(src)
```

Comment:
```suggestion
		new	/obj/item/scalpel(src)
```

Reply:
Why?

Reply:
its off by one, not uniform

Reply:
Blep using tabs on this is hard do to needing to copy past, sorry for the mess up and 400 edits

Reply:
Oh, I misread the suggestion, thanks I guess.